test/metabase/query_processor_test/breakout_test.clj

```diff
@@ -229,8 +229,10 @@
           (mt/with-native-query-testing-context query
             (is (= [[10.0 1] [32.0 4] [34.0 57] [36.0 29] [40.0 9]]
                    (mt/formatted-rows [1.0 int]
-                     (qp/process-query query))))))))
+                     (qp/process-query query))))))))))
 
+(deftest default-binning-test
+  (mt/test-drivers (mt/normal-drivers-with-feature :binning :nested-queries)
     (testing "should be able to use :default binning in a nested query"
```

Comment:
I broke a few mega tests that were like 50 lines into several smaller tests. Easier to debug when I break things. They were logically separate test cases anyway -- they had zero shared state. Generally I think smaller test cases are better, since they're a easier to debug